xmake/scripts/xrepo/envs/zig.lua

```diff
@@ -0,0 +1 @@
+add_requires("zig")
```

Comment:
Alternatively, if there is only one package, it is possible to bind them directly, even without adding any environment. So all these environments are not necessary.

you can run `xrepo env -b zig shell` without this script.